ui.tests/test-module/specs/customfunctionedgedelivery/customfunctionedgedelivery.spec.js

```diff
@@ -0,0 +1,51 @@
+/*
+ *
+ * ADOBE CONFIDENTIAL
+ * ___________________
+ *
+ * Copyright 2024 Adobe Systems Incorporated
+ * All Rights Reserved.
+ *
+ * NOTICE:  All information contained herein is, and remains
+ * the property of Adobe Systems Incorporated and its suppliers,
+ * if any.  The intellectual and technical concepts contained
+ * herein are proprietary to Adobe Systems Incorporated and its
+ * suppliers and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Adobe Systems Incorporated.
+ */
+
+describe("Form to check custom function defined in edge delivery", () => {
+
+    const pagePath = "content/forms/af/core-components-it/samples/customfunctionedgedelivery/customfunctionedgedelivery.html"
+    let toggle_array = [];
+    let formContainer = null
+
+    beforeEach(() => {
+        cy.previewForm(pagePath).then(p => {
+            formContainer = p;
+        })
+        cy.fetchFeatureToggles().then((response) => {
+            if (response.status === 200) {
+                toggle_array = response.body.enabled;
+            }
+        });
+    });
+
+
+
+    it(" fullName field should update on change in first name and last name ", () => {
+        if(toggle_array.includes("FT_FORMS-11269") && toggle_array.includes("FT_FORMS-12423")) {
+            const firstName = Object.entries(formContainer._fields)[0];
```

Comment:
Ideally test should only depend on FT, if the PR change list has the FT in the toggle provider JSON ? Please remove the FT from these test cases

Reply:
added FT's in toggle provider JSON

Reply:
Are these FT's dependent on the feature which is part of this PR ? If not, please remove these from the CL

Reply:
done